src/components/Jam.jsx

```diff
@@ -106,22 +147,21 @@ export default function Jam() {
 
     const body = { destination_addresses: destinations }
 
-    // Todo: Testing toggle is deactivated until https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/1260 is merged.
-    // if (process.env.NODE_ENV === 'development' && useInsecureTestingSettings) {
-    //   body.tumbler_options = {
-    //     addrcount: 3,
-    //     minmakercount: 1,
-    //     makercountrange: [1, 1],
-    //     mixdepthcount: 3,
-    //     mintxcount: 2,
-    //     txcountparams: [1, 1],
-    //     timelambda: 0.1,
-    //     stage1_timelambda_increase: 1.0,
-    //     liquiditywait: 10,
-    //     waittime: 1.0,
-    //     mixdepthsrc: 0,
-    //   }
-    // }
+    if (process.env.NODE_ENV === 'development' && useInsecureTestingSettings && !deactivateTestingToggle) {
+      body.tumbler_options = {
+        addrcount: 3,
+        minmakercount: 1,
+        makercountrange: [1, 1],
```

Comment:
```suggestion
        makercountrange: [1, 0],
```

https://github.com/JoinMarket-Org/joinmarket-clientserver/blob/50f5228498c670258740cd3cab485bcb39e177a3/jmclient/jmclient/cli_options.py#L317-L318

Second param is `spread`. We want exactly one maker in regtest.